templates/csharp/custom-copilot-rag-api/.gitignore

```diff
@@ -0,0 +1,30 @@
+# TeamsFx files
```

Comment:
the template folder name should be custom-copilot-custom-api like the name in js/python.
The fxcore will use the folder name for scafflolding.

Reply:
OK, sure, will update